src/annotation.ts

```diff
@@ -116,36 +131,36 @@ export class Annotation extends Modifier {
     this.setWidth(Flow.textWidth(text));
   }
 
-  getCategory() {
+  getCategory(): string {
     return Annotation.CATEGORY;
   }
 
   // Set font family, size, and weight. E.g., `Arial`, `10pt`, `Bold`.
-  setFont(family, size, weight) {
+  setFont(family: string, size: number, weight: string): this {
     this.font = { family, size, weight };
     return this;
   }
 
   // Set vertical position of text (above or below stave). `just` must be
   // a value in `Annotation.VerticalJustify`.
-  setVerticalJustification(just) {
+  setVerticalJustification(just: string | number): this {
```

Comment:
Should this be typed as `just: string | Justify` ?

Reply:
done